pkg/virt-handler/isolation/process_test.go

```diff
@@ -107,3 +108,44 @@ func (p ProcessStub) PPid() int {
 func (p ProcessStub) Executable() string {
 	return p.binary
 }
+
+// This test has been thankfully taken from the repo https://github.com/mitchellh/go-ps/
+func TestFindProcess(t *testing.T) {
+	p, err := FindProcess(os.Getpid())
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+	if p == nil {
+		t.Fatal("should have process")
+	}
+
+	if p.Pid() != os.Getpid() {
+		t.Fatalf("bad: %#v", p.Pid())
+	}
+}
+
+// This test has been thankfully taken from the repo https://github.com/mitchellh/go-ps/
+func TestProcesses(t *testing.T) {
+	// This test works because there will always be SOME processes
+	// running.
+	p, err := Processes()
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	if len(p) <= 0 {
+		t.Fatal("should have processes")
+	}
+
+	found := false
+	for _, p1 := range p {
+		if p1.Executable() == "bazel" {
+			found = true
+			break
+		}
+	}
```

Comment:
@alicefr in the original code, this check is as below:
```go
if p1.Executable() == "go" || p1.Executable() == "go.exe"
```
which failed for me locally when I did `make test`. I guess it is because kubevirt runs the test in a container. So I have modified it to check for `bazel` instead. WDYT?

Reply:
Tests can be run without bazel with `make go-test`, so the test should support both ways.

Reply:
Thanks, I've changed it to:
```go
if p1.Executable() == "bazel" || p1.Executable() == "go" || p1.Executable() == "go.exe"
```